react/tic-tac-toe/src/app/components/AuthRoute/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { LOGIN } from '../../../constants/routes';
+import { TOKEN } from '../../../constants/auth';
+
+class AuthRoute extends Component {
+  componentDidMount() {
+    const token = localStorage.getItem(TOKEN);
+    if (this.isPrivate && !token) {
```

Comment:
This case is ok, but what about the case where !isPrivate and token is truthy? Should i render the login screen when im already logged?